### PR TITLE
rename INSTALL.txt -> INSTALL.md

### PR DESCRIPTION
**Pull request by davelab6**
- rename INSTALL.txt -> INSTALL.md so that Github will render the file's markup.
- correct tab to spaces
- MANIFEST.in: Update INSTALL.{txt,md}
- correct a tab to 4 spaces

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -54,7 +54,8 @@ In particular, https://github.com/trufont/defcon/tree/python3-ufo3
 
 Homepage: http://www.pyinstaller.org
 
-    pip install PyInstaller ;
+    sudo easy_install pip ;
+	sudo pip install PyInstaller ;
```
Comment (adrientetar) [inline INSTALL.md]: Doesn't homebrew install pip? e.g. https://github.com/Homebrew/homebrew/issues/33897

Comment (davelab6) [inline INSTALL.md]: It appears so :)

**Comment by davelab6** [comment]
@adrientetar Would you prefer this single commit, or a couple of commits so its clearer what changed?

**Comment by adrientetar** [comment]
@davelab6 Can you rebase this and put it into a single commit?

**Comment by felipesanches** [comment]
Dave is AFK.
I rebased it and opened a new PR for this here:
https://github.com/trufont/trufont/pull/89

**Comment by davelab6** [comment]
Closing as Felipe submitted a new one

**Comment by felipesanches** [comment]
@davelab6 aha! I see you had a nice flight :-D

**Comment by davelab6** [comment]
Actually I'm still on the flight; it has wifi :)

**Pull request closed without merging**